content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -225,7 +225,7 @@ A number of markers were added in Kubernetes 1.16 and 1.17, to allow API
 developers to describe the merge strategy supported by lists, maps, and
 structs. These markers can be applied to objects of the respective type,
 in Go files or in the OpenAPI schema definition of the
-[CRD](/docs/reference/generated/kubernetes-api/{{< param "version" >}}#jsonschemaprops-v1-apiextensions-k8s-io):
+{{< api-reference page="common-definitions/json-schema-props" text="CRD" >}}:
```

Comment:
how does 'JSONSchemaProps' become a common definition?

This grouping of resources/definitions is so ad hoc ...

Reply:
It has been moved to another page in the last version. It will be updated in the next commit after I rebased the PR correctly